codegenerator/cli/templates/static/codegen/src/eventFetching/chainWorkers/HyperFuelWorker.res

```diff
@@ -316,7 +317,12 @@ module Make = (
     )
 
     //Instantiate each time to add new registered contract addresses
-    let recieptsSelection = getRecieptsSelection(~contractAddressMapping, ~shouldApplyWildcards)
+    let recieptsSelection = if isPreRegisteringDynamicContracts {
```

Comment:
Hmm, is there no way to catch this at codegen time?  I guess the user will get notified as soon as it starts. So, fine.

Reply:
nvm - it is temprorary too.

Reply:
No since the config is set with the js API